src/pages/cli-legacy/graphql-transformer/storage.mdx

```diff
@@ -1,10 +1,14 @@
 export const meta = {
   title: `GraphQL transform and Storage`,
-  description: `The GraphQL Transform, Amplify CLI, and Amplify Library make it simple to add complex object support with Amazon S3 to an application.`,
+  description: `The GraphQL Transform, Amplify CLI, and Amplify Library make it simple to add complex object support with Amazon S3 to an application.`
 };
 
 The GraphQL Transform, Amplify CLI, and Amplify Library make it simple to add complex object support with Amazon S3 to an application.
 
+<Callout warning>
+  Note: Complex objects are not supported by DataStore-enabled GraphQL APIs.
```

Comment:
```suggestion

  Note: Complex objects are not supported by DataStore-enabled GraphQL APIs.
  
```

Reply:
added some extra space to help rendering

Reply:
Sweet, thanks Stephen!